Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,4381 @@
+
+    Args:
+        client (JiraBaseClient): The Jira client
+        issue (Dict[str, Any]): The issue to retrieve its subtasks and linked issues.
+        expand_links (bool, optional): Whether to retrieve the subtasks and linked issues. Defaults to False.
+
+    Returns:
+        List[Dict[str, Any]]:  A list of subtasks and linked issues corresponding to the given issue.
+    """
+    responses: List[Dict[str, Any]] = []
+    if expand_links:
+        responses.extend(
+            client.get_issue(full_issue_url=sub_task.get('self', ''))
+            for sub_task in issue.get('fields', {}).get('subtasks', [])
+        )
+        for linked_issues in issue.get('fields', {}).get('issuelinks', []):
+            if inward_issue := linked_issues.get('inwardIssue'):
+                responses.append(client.get_issue(full_issue_url=inward_issue.get('self', '')))
+            elif outward_issue := linked_issues.get('outwardIssue'):
+                responses.append(client.get_issue(full_issue_url=outward_issue.get('self', '')))
+    return responses
+
+
+def create_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> CommandResults:
+    """This command is in charge of creating a new issue.
+
+    Args:
+        client (JiraBaseClient): The Jira client.
+        args (Dict[str, str]): The arguments supplied by the user.
+
+    Returns:
+        CommandResults: CommandResults to return to XSOAR.
+    """
+    if 'project_name' in args:
+        args['project_id'] = get_project_id_from_name(client=client, project_name=args['project_name'])
+    issue_fields = create_issue_fields(client=client, issue_args=args,
+                                       issue_fields_mapper=client.ISSUE_FIELDS_CREATE_MAPPER)
+    res = client.create_issue(json_data=issue_fields)
+    outputs = {'Id': res.get('id', ''), 'Key': res.get('key', '')}
+    markdown_dict = outputs | {'Ticket Link': res.get('self', ''),
+                               'Project Key': res.get('key', '').split('-')[0]}
+    return CommandResults(
+        outputs_prefix='Ticket',
+        outputs=outputs,
+        outputs_key_field='Id',
+        readable_output=tableToMarkdown(name=f'Issue {outputs.get("Key", "")}', t=markdown_dict),
+        raw_response=res
+    )
+
+
+def edit_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> CommandResults:
+    """This command is in charge of editing an existing issue.
+
+    Args:
+        client (JiraBaseClient): The Jira client.
+        args (Dict[str, str]): The arguments supplied by the user.
+
+    Raises:
+        DemistoException: If both the status and transition are provided.
+        DemistoException: If neither an issue id nor a key was supplied.
+
+    Returns:
+        CommandResults: CommandResults to return to XSOAR.
+    """
+    issue_id_or_key = args.get('issue_id', args.get('issue_key', ''))
+    if not issue_id_or_key:
+        raise DemistoException(ID_OR_KEY_MISSING_ERROR)
```

Comment:
what happens if we have both id and key? and they are referring to different issues by mistake?